lib/msal-core/src/DatabaseCache.ts

```diff
@@ -0,0 +1,65 @@
+interface IDBOpenDBRequestEvent extends Event {
```

Comment:
Can you add some context here on the design and the flexibility you are trying to add ?

Reply:
Yeah, will do.